doc/hands_on/hands_on_dynamodb_vs.md

```diff
@@ -0,0 +1,229 @@
+1. Upload the adapter to the BucketFS:
+    ``` shell script
+   curl -I -X PUT -T dynamodb-virtual-schemas-adapter-dist-0.4.0.jar http://w:writepw@<YOUR_DB_IP>:2580/default/dynamodb-virtual-schemas-adapter-dist-0.4.0.jar
+   ```
+1. Create a schema to hold the adapter script:
+    ```sql
+   CREATE SCHEMA ADAPTER;
+   ```
+1. Create the Adapter Script:
+    ```sql
+    CREATE OR REPLACE JAVA ADAPTER SCRIPT ADAPTER.DYNAMODB_ADAPTER AS
+       %scriptclass com.exasol.adapter.RequestDispatcher;
+       %jar /buckets/bfsdefault/default/dynamodb-virtual-schemas-adapter-dist-0.4.0.jar;
+    /
+    ```
+1. Create UDF:
+    ```sql
+    CREATE OR REPLACE JAVA SET SCRIPT ADAPTER.IMPORT_DOCUMENT_DATA(
+      DOCUMENT_FETCHER VARCHAR(2000000),
+      REMOTE_TABLE_QUERY VARCHAR(2000000),
+      CONNECTION_NAME VARCHAR(500))
+      EMITS(...) AS
+        %scriptclass com.exasol.adapter.dynamodb.ImportDocumentData;
+        %jar /buckets/bfsdefault/default/dynamodb-virtual-schemas-adapter-dist-0.4.0.jar;
+    /
+   ```
+   
+## Create a Mapping Definition
+Now we need to tell the adapter how to map the DynamoDB documents to Exasol tables.
+For that we create a file with the [Exasol Document Mapping Language](../gettingStartedWithSchemaMappingLanguage.md).
+
+You can create the file wherever you want. We will later upload it to BucketFS. 
+
+`firstMapping.json`:
+```json
+{
+  "$schema": "../../main/resources/mappingLanguageSchema.json",
+  "srcTable": "Books",
+  "destTable": "BOOKS",
+  "description": "Mapping for the Books table",
+  "mapping": {
+    "fields": {
+      "Title": {
+        "toVarcharMapping": {
+          "varcharColumnSize": 254
+        }
+      }
+    }
+  }
+}
+``` 
+
+Now upload the mapping to BucketFS:
+
+```shell script
+curl -I -X PUT -T firstMapping.json http://w:writepw@<YOUR_DB_IP>:2580/default/mappings/firstMapping.json
+```
+
+## Create Virtual Schema
+
+Nwo we can create the Virtual Schema.
+
+Steps:
+
+1. Create a connection to DynamoDB
+    * For DynamoDB on AWS use:
+       ```sql
+      CREATE CONNECTION DYNAMO_CONNECTION
+         TO 'aws:<REGION>'
+         USER '<AWS_ACCESS_KEY_ID>'
+         IDENTIFIED BY '<AWS_SECRET_ACCESS_KEY>';
+      ```
+    * For a local DynamoDB use:
+       ```sql
+      CREATE CONNECTION DYNAMO_CONNECTION
+          TO 'http://<YOUR_DYNAMODB_IP>:8000'
+          USER 'fakeMyKeyId'
+          IDENTIFIED BY 'fakeSecretAccessKey';
+      ```
+      
+2. Create Virtual Schema:
+    ```sql
+   CREATE VIRTUAL SCHEMA DYNAMODB_TEST USING ADAPTER.DYNAMODB_ADAPTER WITH
+       CONNECTION_NAME = 'DYNAMO_CONNECTION'
+       SQL_DIALECT     = 'DYNAMO_DB'
+       MAPPING         = '/bfsdefault/default/mappings/firstMapping.json';
+   ```
+ 
+ ## First Results
+ 
+ Now you can browse the newly created table `DYNAMODB_TEST.BOOKS`:
+ 
+ ![alt text](result.png "Virtual Schema result")
+ 
+ ## Next Steps
+ In the next part of this series we will show how to create more complex mappings.
+ 
```

Comment:
Please, check the English spelling of the whole article yourself, using some tools. I usually use `grammarly`